source/develop/developer-guide/engine/hosted-engine-invalid-bond-modes.md

```diff
@@ -0,0 +1,27 @@
+---
+title: Hosted-Engine invalid bond modes
+authors: Ido Rosenzwig
+---
+
+# Hosted-Engine invalid bond modes
+
+Some bond modes (0, 5, 6) cause problems in VM networks.
+Therefore, when the user has to select network device, during the Hosted-engine deployment,
+bonds that are configured in these modes cannot be selected.
+
+## Enable invalid bond modes
+
+If one wish to configure Hosted-Engine with unsupported bond mode, it can be done by doing the following:               
```

Comment:
perhaps use "force" or "enforce" or something like that, that makes it clear you are doing something risky.